### ISSUE TITLE
`svelte-kit package` doesn't remove `lang="ts"` attribute from `<script>` tags

### ISSUE THREAD

**Issue by GrygrFlzr** [sveltejs/kit#1843]
**Describe the bug**
`svelte-kit package` on a TypeScript project doesn't remove `lang="ts"` attribute from `<script>` tags. The code is transformed from TS to JS, though.

**To Reproduce**
```bash
npm init svelte@next
# choose demo
# choose typescript
# choose whatever for prettier/eslint
npm i
npm i -D svelte2tsx
npx svelte-kit package
```

Produces the following `package/Counter/index.svelte` script tag:
```svelte
<script lang="ts">import { spring } from 'svelte/motion';
let count = 0;
const displayed_count = spring();
$: displayed_count.set(count);
$: offset = modulo($displayed_count, 1);
function modulo(n, m) {
    // handle negative numbers
    return ((n % m) + m) % m;
}
</script>
```

**Expected behavior**
Remove the `lang="ts"` attribute from `<script>` tags.

**Information about your SvelteKit Installation:**

<details><summary>Diagnostics</summary>

```
  System:
    OS: Windows 10 10.0.22000
    CPU: (16) x64 AMD Ryzen 7 3700X 8-Core Processor
    Memory: 9.93 GB / 31.93 GB
  Binaries:
    Node: 14.16.0 - C:\program files\nodejs\node.EXE
    Yarn: 1.22.10 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 6.14.13 - C:\program files\nodejs\npm.CMD
  Browsers:
    Chrome: 91.0.4472.124
    Edge: Spartan (44.22000.1.0), Chromium (91.0.864.59)
    Internet Explorer: 11.0.22000.1
  npmPackages:
    @sveltejs/kit: next => 1.0.0-next.122
    svelte: ^3.34.0 => 3.38.3
```
</details>

**Severity**
Annoyance - requires manual removal for proper publishing.

**Comment by dummdidumm** [comment]
Issue lies within `svelte-preprocess`, there's an issue for that: https://github.com/sveltejs/svelte-preprocess/issues/260 . Closing as a duplicate.

**Comment by probablykasper** [comment]
@dummdidumm This issue is making my package unusable. Would it be fair to reopen this and require it to be fixed before SvelteKit 1.0?

<details>
<summary>Error log from using package</summary>

```
5:22:33 PM [vite] Internal server error: Expected newline.
  ╷
2 │ .date-time-picker {
  │                   ^
  ╵
  ../svelte-date-picker/package/DatePicker.svelte 2:19  root stylesheet
  Plugin: vite-plugin-svelte
  File: /Users/kasper/dev/git/svelte-date-picker/package/DatePicker.svelte
  1  |  <script lang="ts">
  2  |    import { getMonthLength, getWeeks } from './utils/date-utils'
     |                     ^
  3  |    export let value = new Date()
  4  |    export let visible = true
      at Object._newRenderError (/Users/kasper/dev/git/kryp/node_modules/sass/sass.dart.js:2203:19)
      at Object._wrapException (/Users/kasper/dev/git/kryp/node_modules/sass/sass.dart.js:2023:16)
      at _render_closure1.call$2 (/Users/kasper/dev/git/kryp/node_modules/sass/sass.dart.js:82163:21)
      at _RootZone.runBinary$3$3 (/Users/kasper/dev/git/kryp/node_modules/sass/sass.dart.js:27566:18)
      at _FutureListener.handleError$1 (/Users/kasper/dev/git/kryp/node_modules/sass/sass.dart.js:26096:21)
      at _Future__propagateToListeners_handleError.call$0 (/Users/kasper/dev/git/kryp/node_modules/sass/sass.dart.js:26395:49)
      at Object._Future__propagateToListeners (/Users/kasper/dev/git/kryp/node_modules/sass/sass.dart.js:12483:77)
      at _Future._completeError$2 (/Users/kasper/dev/git/kryp/node_modules/sass/sass.dart.js:26241:9)
      at _AsyncAwaitCompleter.completeError$2 (/Users/kasper/dev/git/kryp/node_modules/sass/sass.dart.js:25910:12)
      at Object._asyncRethrow (/Users/kasper/dev/git/kryp/node_modules/sass/sass.dart.js:12286:17)
```

</details>
